library/alloc/src/vec/mod.rs

```diff
@@ -1991,15 +1991,17 @@ impl<T, A: Allocator> Vec<T, A> {
     #[stable(feature = "rust1", since = "1.0.0")]
     #[rustc_confusables("push_back", "put", "append")]
     pub fn push(&mut self, value: T) {
+        // Inform LLVM that the length does not change across grow_one().
```

Comment:
This isn't specific to LLVM...
```suggestion
        // Inform codegen that the length does not change across grow_one().
```